ducktape/cluster/vagrant.py

```diff
@@ -12,9 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .cluster import Cluster, ClusterSlot
-from .json import JsonCluster
+from __future__ import absolute_import
 
+from .json import JsonCluster
+from .cluster import Cluster, ClusterSlot
+from ducktape.command_line.config import ConsoleConfig
+import os
+import json
```

Comment:
Hm I see why you had to do `from __future__ import absolute_import` here...

I wonder if we should rename our `cluster.json` module to `cluster.json_cluster` to avoid  this kind of collision?